src/Shared/FileMatcher.cs

```diff
@@ -31,6 +31,8 @@ internal class FileMatcher
         private static readonly char[] s_wildcardCharacters = { '*', '?' };
         private static readonly char[] s_wildcardAndSemicolonCharacters = { '*', '?', ';' };
 
+        private static readonly string[] s_propertyReferences = { "$(", "@(" };
```

Comment:
```suggestion
        private static readonly string[] s_propertyAndItemReferences = { "$(", "@(" };
```

Items are referred to with an `@` and properties with `$`.

Reply:
Thanks, I just copied from the old name of the method. I adjusted the naming now to these.